com.unity.postprocessing/PostProcessing/Editor/Unity.Postprocessing.Editor.asmdef

```diff
@@ -1,9 +1,11 @@
 {
     "name": "Unity.Postprocessing.Editor",
+    "rootNamespace": "",
     "references": [
-        "Unity.Postprocessing.Runtime"
+        "Unity.Postprocessing.Runtime",
+        "Unity.XR.Management",
```

Comment:
Do we need these new deps here ? Can we avoid them ?

Reply:
I'll double check if we can avoid new dependency.

Reply:
We need these references, otherwise the code that query the XR Management module can't work.

But it does not add a strict dependency to XR Management extension, it will only use it if the package is added to the project by the user (detected in code via the preproc `#if ENABLE_XR_MODULE && XR_MANAGEMENT_4_0_1_OR_NEWER`), and if its not there, the part of the code that require it is disabled.